conda-store-server/conda_store_server/api.py

```diff
@@ -82,6 +83,176 @@ def update_namespace(
     return namespace
 
 
+# v2 API
+def update_namespace_metadata(
+    db,
+    name: str,
+    metadata_: Dict[str, Any] = None,
+):
+    namespace = get_namespace(db, name)
+    if namespace is None:
+        raise ValueError(f"Namespace='{name}' not found")
+
+    if metadata_ is not None:
+        namespace.metadata_ = metadata_
+
+    return namespace
+
+
+# v2 API
+def get_namespace_roles(
+    db,
+    name: str,
+):
+    """Which namespaces can access namespace 'name'?"""
+    namespace = get_namespace(db, name)
+    if namespace is None:
+        raise ValueError(f"Namespace='{name}' not found")
+
+    nrm = aliased(orm.NamespaceRoleMappingV2)
```

Comment:
Can we please have more descriptive variable names here? Readability is part of maintainability.

Reply:
In general, I agree. But not here. I deliberately use short names so that queries don't get split up over multiple lines by the linter, which would be hard to read. All these DB functions are very small, so there's no concern of this clashing with something else causing confusion. I couldn't think of a short name for NamespaceRoleMapping other than using the first letters.